.devcontainer/Dockerfile

```diff
@@ -313,6 +313,6 @@ RUN go get -v golang.org/x/tools/gopls
 #### Update shared library configuration
 RUN ldconfig -v
 
-RUN ln -s /magma/experimental/bazel-base/bazelrcs/devcontainer.bazelrc /etc/bazelrc
+RUN ln -s /workspaces/magma/experimental/bazel-base/bazelrcs/devcontainer.bazelrc /etc/bazelrc
```

Comment:
Use the MAGMA_ROOT environment variable instead

Reply:
ah good point!